README.md

```diff
@@ -1,2 +1,20 @@
-# silver-express-api
-Silver Express Api
+![Silver App Logo](/src/assets/images/silver-logo_white_font-color.png "Silver App Logo")
+# Silver Express API
+
+Silver Express is a taxicab company.
+
+## Run the project
+
+To run the project:
+
+```sh
+npm start
+```
+
+## Completed Tasks
+
+### Installed dependencies
+- auth0_flutter: ^1.2.1
```

Comment:
Are these dependencies necessary in the backend?

Reply:
Remained from the README.md model of the app. Only the dependencies that were used in the API should be added, this dependency does not exist in the API.